specs/altair/sync-protocol.md

```diff
@@ -237,13 +269,12 @@ def validate_light_client_update(store: LightClientStore,
 
 ```python
 def apply_light_client_update(store: LightClientStore, update: LightClientUpdate) -> None:
-    active_header = get_active_header(update)
-    finalized_period = compute_sync_committee_period(compute_epoch_at_slot(store.finalized_header.slot))
-    update_period = compute_sync_committee_period(compute_epoch_at_slot(active_header.slot))
-    if update_period == finalized_period + 1:
+    store_period = compute_sync_committee_period(compute_epoch_at_slot(store.finalized_header.slot))
+    finalized_period = compute_sync_committee_period(compute_epoch_at_slot(update.finalized_header.slot))
```

Comment:
So the old `finalized_period` is now called `store_period`, and the `update`'s sync committee period is now called `finalized_period`. IMO `finalized_period` is a bit unclear here.

What do you think about:
- renaming `finalized_period` to `update_finalized_period`
- remaming `attested_period` to `update_attested_period` (just for aligning with `update_finalized_period`)

Reply:
There are three sync committee periods in an update:

- `finalized_period` --> based on `finalized_header.slot`; used to infer when a sync committee period has finalized
- `attested_period` --> based on `attested_header.slot`; can advance sync committee once `finalized_period` same
- `signature_period` --> based on `signature_slot`; indicates which sync committee actually signed the message

On top of that, there is `store_period` which tracks the sync committee period to which the `LightClientStore`'s `current_sync_committee` is linked. This is tracked as part of `store.finalized_header.slot`, which normally matches the actual finalized header, but may be set to a later slot in case of forced updates. `store.finalized_header` reflects the sync committee period that the `LightClientStore` is locked to and cannot revert back from.

The variables are named consistently, there should be no more `finalized_period` usage reflecting to the `store_period`.

So, the full suggestion is this one?
- `finalized_period` --> `update_finalized_period`
- `attested_period` --> `update_attested_period`
- `signature_period` --> `update_signature_period`
- `store_period` --> `store_period`

Reply:
> So, the full suggestion is this one?

Yes, correct!

Reply:
b7a0e532e476b3bbc8ffdebdaf06c172b7b911c4